app/src/main/kotlin/com/antosik/benchproject/app/favourite/movies/view/FavouriteMoviesFragment.kt

```diff
@@ -0,0 +1,67 @@
+package com.antosik.benchproject.app.favourite.movies.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Text
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import androidx.compose.ui.unit.dp
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.antosik.benchproject.app.common.state.UIState.Empty
+import com.antosik.benchproject.app.common.state.UIState.Loading
+import com.antosik.benchproject.app.common.state.UIState.Success
+import com.antosik.benchproject.app.common.theme.BenchProjectTheme
+import com.antosik.benchproject.app.favourite.movies.viewModel.FavouriteMoviesViewModel
+import com.antosik.benchproject.app.popular.movies.entity.Movie
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class FavouriteMoviesFragment : Fragment() {
+
+    private val favouriteMoviesViewModel: FavouriteMoviesViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ) = ComposeView(requireContext()).apply {
+        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+        setContent {
+            BenchProjectTheme {
+                Column(
+                    modifier = Modifier
+                        .verticalScroll(rememberScrollState())
+                        .padding(bottom = 58.dp),
```

Comment:
why such a strange padding value :D ? i am just curious :D

Reply:
Because of the bottom navigation in XML and other Compose view, I have something similar. :)

### Examples: 
**XML** popular_movies_fragment.xml
![image](https://github.com/Nataniel-Antosik/Bench_Project/assets/57491794/fa0448f9-e241-4fd0-ac7a-af235edf8361)

**Compose** MoreScreen.Kt
![image](https://github.com/Nataniel-Antosik/Bench_Project/assets/57491794/fa6b71ba-8901-4655-b1a2-9fa2e01a71ec)

And yes now I see on the main screen elements are a little higher than on **MoreScreen** (30.dp different)